hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -1430,6 +1432,10 @@ public static SCMMetrics getMetrics() {
     return metrics == null ? SCMMetrics.create() : metrics;
   }
 
+  public static SCMPerformanceMetrics getPerfMetrics() {
```

Comment:
You will also need to create/register the newly added metrics instance separately during server startup. Please refer: `SCMMetrics.create()`. 

This would only register the metrics system when the method `getPerfMetrics()` is called.